src/gluonts/ev/metrics.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass
+from functools import partial
+from typing import Optional
+
+import numpy as np
+
+from .api import (
+    DerivedMetricEvaluator,
+    MetricEvaluator,
+    StandardMetricEvaluator,
+)
+from .aggregations import Mean, Sum
+from .stats import (
+    absolute_error,
+    absolute_label,
+    absolute_percentage_error,
+    absolute_scaled_error,
+    coverage,
+    quantile_loss,
+    scaled_interval_score,
+    squared_error,
+    symmetric_absolute_percentage_error,
+)
+
+
+def mean_absolute_label(axis: Optional[int] = None) -> StandardMetricEvaluator:
+    return StandardMetricEvaluator(
+        map=absolute_label,
+        aggregate=Mean(axis=axis),
+    )
+
+
+def sum_absolute_label(axis: Optional[int] = None) -> StandardMetricEvaluator:
+    return StandardMetricEvaluator(
+        map=absolute_label,
+        aggregate=Sum(axis=axis),
+    )
+
+
+@dataclass
+class SumAbsoluteError:
+    forecast_type: str = "0.5"
+
+    def __call__(self, axis: Optional[int] = None) -> StandardMetricEvaluator:
+        return StandardMetricEvaluator(
+            map=partial(absolute_error, forecast_type=self.forecast_type),
+            aggregate=Sum(axis=axis),
+        )
+
+
+@dataclass
+class MeanSquaredError:
```

Comment:
Shouldn't we name this `MSE`?

Let's have abbreviated names for metrics.

Reply:
Sure. As not every metric has a common abbreviation, the metrics would be:

mean_absolute_label
sum_absolute_label
SumAbsoluteError
MSE
SumQuantileLoss
Coverage
MAPE
SMAPE
MSIS
MASE
ND
RMSE
NRMSE
WeightedSumQuantileLoss

Does this look good to you or can some names still be improved?

Reply:
Looks good to me.